src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/RC2.cs

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics;
+
+using Internal.Cryptography;
+
+namespace System.Security.Cryptography
+{
+    public abstract class RC2 : SymmetricAlgorithm
```

Comment:
RC2CryptoServiceProvider has `public bool UseSalt { get; set; }`.  Do we want/need that on the base class?  (I'm not familiar enough with RC2 to know.)